home/templates/home/explore.html

```diff
@@ -0,0 +1,47 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+<script type="text/javascript" src="{% static 'home/js/explore_page_js.js' %}"></script>
+<link rel="stylesheet" type="text/css" href="{% static 'home/CSS/explore-page.css' %}">
+<body class="landing-bg">
```

Comment:
About the class landing-bg. it's a very light color used for the landing page.
I think you should change it to the 'main-bg' class so it will match the design in the Wireframe, and have
identical bg color to the display question page

Reply:
@liornoy 
Done